src/Akka.Persistence.Azure.API.Tests/verify/ApiSpecs.ApproveCore.DotNet.verified.txt

```diff
@@ -129,6 +130,11 @@ namespace Akka.Persistence.Azure.Journal
         public Akka.Persistence.Azure.Journal.AzureTableStorageJournalSetup Get(string journalId) { }
         public void Set(string journalId, Akka.Persistence.Azure.Journal.AzureTableStorageJournalSetup setup) { }
     }
+    public class DatabaseTransactionException : System.Exception
```

Comment:
Probably needs a more specific name to make it clear that this is coming from the Akka.Persistence.Azure plugin, not some generic thing